source/billing-data.txt

```diff
@@ -6,48 +6,138 @@ Billing Data
---------
-
-The following examples <perform this action> using |service|
-:ref:`tools for automation <arch-center-automation>`.
-
-These examples also apply other recommended configurations, including:
+Tracking and managing cloud spending can be difficult in a large
+organization. While :ref:`cost-saving configurations
+<arch-center-cost-saving-config>` provides strategies to proactively
+save money, {+service+} also provides cloud spend governance tools for
+you to:
 
-.. tabs::
+- Categorize {+service+} resources based on your organization's billing needs.
+- Leverage billing data to visualize and understand your {+service+} spending.
+- Receive notifications when Atlas spending exceeds a specified threshold.
+- Pull billing data programmatically to integrate with your FinOps tools.
 
-   .. tab:: Dev and Test Environments
-      :tabid: devtest
+On this page, learn how to use built-in {+service+} tools and
+{+service+} billing data to track, manage, and optimize your cloud
+spending.
 
-      .. include:: /includes/shared-settings-clusters-devtest.rst
-
-   .. tab:: Staging and Prod Environments
-      :tabid: stagingprod
-
-      .. include:: /includes/shared-settings-clusters-stagingprod.rst
-
-.. tabs::
-
-   .. tab:: CLI
-      :tabid: cli
-
-      Content here
-
-   .. tab:: Terraform
-      :tabid: Terraform
-
-      Content here
+{+service+} Features and Best Practices for Billing Data
+--------------------------------------------------------
 
+Billing Dashboards
+~~~~~~~~~~~~~~~~~~
+
+|charts| integreates seemlessly with {+service+}. You can visualize your
+billing data in a billing dashboard to help you optimize your |service|
+spending. Billing dashboards contain prebuilt charts that help you
+monitor your {+service+} usage in an organization across different
+categories and periods of time.
+
+.. figure:: /includes/images/atlas-billing-dashboard.png
+   :figwidth: 750px
+   :alt: Atlas Billing Dashboard example.
+
+To create or manage a billing dashboard, see :ref:`billing-dashboards`.
+
+Apply Resource Tags
+~~~~~~~~~~~~~~~~~~~
+
+The {+atlas-admin-api+} provides a |rest|\ful interface that allows you
+to retrieve all your pending invoices with the :oas-atlas-op:`Return All
+Pending Invoices for One Organization </listPendingInvoices>` |api|
+endpoint. The endpoints response body contains the
+``results.lineItems.tags`` field, which you can apply to your resources.
+
+Resource tags can help you categorize your spending and allow you to
+feed the |api| data to reports and external FinOps tools they already
+use. To learn more, see :ref:`configure-resource-tags`.
+
+Billing Quota Management
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+|service| offers real-time notifications when spending nears or exceeds
+set thresholds. You can customize these alerts to be project-specific or
+team-specific, allowing for precise monitoring and control over
+different areas of cloud expenditure. To configure billing alerts, see
+:ref:`configure-alerts`.
```

Comment:
I think this Billing Quota Management section belongs under the Cost-Savings Configurations instead